addons/binding/org.openhab.binding.network.test/src/test/java/org/openhab/binding/network/internal/PresenceDetectionTest.java

```diff
@@ -41,19 +42,27 @@
 @RunWith(MockitoJUnitRunner.class)
 public class PresenceDetectionTest {
     static long CACHETIME = 2000L;
+    @Mock
     NetworkUtils networkUtils;
     PresenceDetection presenceDetection;
```

Comment:
A little offtopic I think the overview will be better if you, put this field below all @Mock initialisations and maybe call it subject, 😄

Reply:
Yup sounds like a good practice